docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Ranking/LightGbm.cs

```diff
@@ -27,39 +32,44 @@ public static void Example()
             // Train the model.
             var model = pipeline.Fit(trainingData);
 
-            // Create testing data. Use different random seed to make it different from training data.
-            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed:123));
+            // Create testing data. Use different random seed to make it different
+			// from training data.
+            var testData = mlContext.Data.LoadFromEnumerable(
+                GenerateRandomDataPoints(500, seed:123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Take the top 5 rows.
-            var topTransformedTestData = mlContext.Data.TakeRows(transformedTestData, 5);
+            var topTransformedTestData = mlContext.Data.TakeRows(
+                transformedTestData, 5);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(topTransformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data.CreateEnumerable<Prediction>(
+                topTransformedTestData, reuseRowObject: false).ToList();
 
             // Print 5 predictions.
             foreach (var p in predictions)
                 Console.WriteLine($"Label: {p.Label}, Score: {p.Score}");
 
             // Expected output:
-            //   Label: 5, Score: 2.195333
-            //   Label: 4, Score: 0.2596574
-            //   Label: 4, Score: -2.168355
-            //   Label: 1, Score: -3.074823
-            //   Label: 1, Score: -1.523607
+            //   Label: 5, Score: 2.493263
```

Comment:
Did these numbers change for the sample?

Reply:
Yeah they must have changed from a more recent pull.

Reply:
@codemz Is that right? Isn't the same data / same seed / same algorithm?

Reply:
Ok, I see that the sample is using random data points. Thanks